src/pages/[slug]/[ie]/index.tsx

```diff
@@ -458,8 +465,15 @@ const ObjectDetailPage: NextPage = () => {
 								</p>
 								<div className="u-pb-24 p-object-detail__actions">
 									<Button
+										disabled={!!metaDataExportError || !metadataExport}
 										className="p-object-detail__export"
 										iconStart={<Icon name="export" />}
+										onClick={() => {
+											save(
+												metadataExport,
+												`${mediaInfo?.name || 'metadata'}.xml`
```

Comment:
use lodash kebab case to avoid special chars/spaces in filenames
```
kebabCase('Foo Bar');
// => 'foo-bar'
```
https://lodash.com/docs/4.17.15#kebabCase